### PR TITLE
Property methods updates

### PR DESCRIPTION
**Pull request by Saksham20**
All the classes of SegmentationExtractor: 1) sima, 2) cnmfe 3)extract 4)suite2p, 5) nwb 6) numpy 
are changed here. 
There are a number of properties which were redundant since there are get methods for each. These were defined in the base class by Allessio so I have removed them from here. 

Semantically: 
1. Properties are removed here since they are in the base class `SegmentationExtractor`.
2. SOme methods and properties you will see are not removed by renamed: matching those renamed in the base class abstract methods in master branch. 

Note: 
There are a few minor additional changes in Suite2p which are there since now we have agreed to have only 1 plane enforced.

### CODE REVIEW

**Comment by bendichter** [comment]
@Saksham20 the proper way to link issues to PRs is to write "fix #25" in the description of the PR. Can you please change it?

File: roiextractors/extractors/numpyextractors/numpyextractors.py [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import numpy as np
 from pathlib import Path
-from ...segmentationextractor import SegmentationExtractor
-from ...imagingextractor import ImagingExtractor
-from ...extraction_tools import get_video_shape
+from roiextractors import SegmentationExtractor, ImagingExtractor
+from roiextractors.extraction_tools import get_video_shape
```
Comment (bendichter) [inline roiextractors/extractors/numpyextractors/numpyextractors.py:4]: use relative imports

File: roiextractors/extractors/numpyextractors/numpyextractors.py [hunk 2]
```diff
@@ -222,47 +208,31 @@ def roi_locs(self):
...
             roi_idx_ = [j[0] for i, j in enumerate(roi_idx) if i not in ele]
-            return self.roi_locs[:, roi_idx_]
+            return self.roi_locations[:, roi_idx_]
 
     def get_num_frames(self):
-        return self.num_of_frames
-
-    def get_sampling_frequency(self):
-        return self.samp_freq
+        return self._roi_response.shape[1]
```
Comment (bendichter) [inline roiextractors/extractors/numpyextractors/numpyextractors.py:235]: Is this the sampling frequency?

File: roiextractors/extractors/schnitzerextractor/extractsegmentationextractor.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 import h5py
 from roiextractors import SegmentationExtractor
 from lazy_ops import DatasetView
-
+from roiextractors.extraction_tools import _pixel_mask_extractor
```
Comment (bendichter) [inline roiextractors/extractors/schnitzerextractor/extractsegmentationextractor.py:5]: use relative imports

File: roiextractors/extractors/numpyextractors/numpyextractors.py (hunk 2 above)
Comment (Saksham20) [reply roiextractors/extractors/numpyextractors/numpyextractors.py:235]: yes, this is the samp rate of the traces.

Comment (bendichter) [reply roiextractors/extractors/numpyextractors/numpyextractors.py:235]: maybe this is the length of the response, i.e. the number of samples?

Comment (Saksham20) [reply roiextractors/extractors/numpyextractors/numpyextractors.py:235]:
```
def get_num_frames(self):
      return self._roi_response.shape[1]
```
this refers to the number of frames. 
`get_sampling_frequency()` method is in the parent class

Comment (bendichter) [reply roiextractors/extractors/numpyextractors/numpyextractors.py:235]: oh, my fault, I was not reading the line changes correctly

Comment (Saksham20) [reply roiextractors/extractors/numpyextractors/numpyextractors.py:235]: When I open the file mentioned here, it does not show what you're mentioning lol
This is what I see:
image

**Pull request merged**